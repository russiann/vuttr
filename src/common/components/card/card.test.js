import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Button from './card.component';

afterEach(cleanup);

it('renders', () => {
  const {asFragment} = render(<Button>Title</Button>);
  expect(asFragment()).toMatchSnapshot();
});
