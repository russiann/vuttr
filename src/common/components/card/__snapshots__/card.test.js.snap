// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="css-1ithia2-Card e1kioe3z0"
  >
    Title
  </div>
</DocumentFragment>
`;
